### PR TITLE
Test eslint

### PR DESCRIPTION
**Pull request by mikeattara**
I check all js files to ensure that the formatting is consistent across the project

### CODE REVIEW

**Comment by MattCSmith** [comment]
This PR along with https://github.com/zeroDevs/coding_challenge-13/pull/14 are recreating the footer component that was moved into a new directory. 

Could you remove this from the PR, so we dont add files that aren't require :)

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
Comment (MattCSmith) [inline src/components/Footer.js]: This file is no longer located here :)

Comment (mikeattara) [reply src/components/Footer.js]: I deleted the redundant file @MattCSmith

**Comment by aneagoie** [comment]
Can we remove this PR now?

**Comment by mikeattara** [comment]
Yes, I realized that I didn't branch well so my latest PR contains all these changes. Sorry for the inconvenience. @aneagoie

**Comment by mikeattara** [comment]
I am closing this pr now.

**Pull request closed without merging**